.changeset/three-trees-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-editor": minor
+---
+
+Make widgets open by default. Add ability to open/close all widgets on a page externally.
```

Comment:
I'd like to work a slight term change through this PR. The "widget" is what the learner sees. The "widget editor" or simply "editor" is what the content author uses to adjust the settings for a widget.

So, could we say "Make widget editors expanded by default. ..."?

Although we don't use the WB Accordion, I think using the same terms is useful.  They use the term "expanded" and "collapsed" for open/closed.